Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -804,18 +807,13 @@ public bool PublishingStopped
         {
             get
             {
-                lock (m_cache)
+                TimeSpan timeSinceLastNotification = TimeSpan.FromTicks(DateTime.UtcNow.Ticks - Interlocked.Read(ref m_lastNotificationTime));
+                if (timeSinceLastNotification.TotalMilliseconds > m_keepAliveInterval + KeepAliveTimerMargin)
```

Comment:
Same question re: overrun/wraparound, but havent thought this through.

Reply:
ticks won't overrun